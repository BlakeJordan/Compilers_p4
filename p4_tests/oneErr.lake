int a;
bool b;
void fn(){
	a = b;
}
