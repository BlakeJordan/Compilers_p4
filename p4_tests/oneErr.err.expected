4,4: Invalid assignment operation
Type checking failed